packages/react-components/theme-designer/etc/theme-designer.api.md

```diff
@@ -0,0 +1,14 @@
+## API Report File for "@fluentui/theme-designer"
```

Comment:
I know these were all automated but `theme-designer` is technically an app so does this really need the api extractor stuff?

Reply:
Im kinda aware of this , but lemme share more context:
- if it's an app it should not live in packages
- what we need is consistency for our packages automated by tooling (migrate-converged-pkg)
- it "acts" as package (has `main` and exports public API) - app should never exports anything 

hope it makes sense. 

If we are 100% confident that this is an app lets make it so